build.sbt

```diff
@@ -25,14 +25,16 @@ scalacOptions in ThisBuild ++= compilerOptions
 val minJacksonVersion: String = "2.8.9"
 val minJacksonLibs = Seq(
   "com.fasterxml.jackson.core" % "jackson-core" % minJacksonVersion,
-  "com.fasterxml.jackson.core" % "jackson-databind" % minJacksonVersion,
   "com.fasterxml.jackson.core" % "jackson-annotations" % minJacksonVersion,
   "com.fasterxml.jackson.dataformat" % "jackson-dataformat-cbor" % minJacksonVersion
 )
 
-val playJsonVersion = "2.6.9"
+val jacksonDatabind = "com.fasterxml.jackson.core" % "jackson-databind" % "2.9.10.1"
```

Comment:
Why not changing the `minJacksonVersion` ?

Reply:
I have now, thanks for suggesting this 👍

Reply:
I have also just bumped up to the latest version as there was a new vulnerability discovered a few days ago (with version 2.9.10.1)